src/service/ampdoc-impl.js

```diff
@@ -754,16 +754,22 @@ export class AmpDoc {
   }
 
   /**
-   * Allow one tracking iframe for each amp doc. Caller need to handle user
-   * error when registeration returns false
+   * Attempt to register a service that is allowed/required one for each ampdoc.
+   * Caller need to handle user error when registration returns false.
+   *
+   * Please list your service name below:
```

Comment:
I don't want to add extra bytes to the runtime by keeping an enum list. But I do want to keep such a list somewhere for reference so that we don't end up with identical service name. Let me know if there's better way to address : )

Reply:
Enums get eliminated if unused, and the `Enum.FOO` reference will be rewritten with whatever the value is if it's used. Enums are basically 0 cost. 😉

Reply:
That's cool! Switched to enum